README.md

```diff
@@ -233,7 +223,6 @@ Small note: If editing the Readme, please conform to the [standard-readme](https
 - [stl2pov](http://rsmith.home.xs4all.nl/software/py-stl-stl2pov.htmltool) to convert .stl to .pov, and then render via [PovRay.org](http://povray.org)
 - [P3D](https://github.com/D1plo1d/p3d) STL/AMF/OBJ viewer
 
-
 That's all for now,
 
-Rene K. Mueller & contributors
+Rene K. Mueller & JSCAD Organization
```

Comment:
Not for 'ego' things at all , but should we Include our names (Jeff, Mark etc) here as well to reflect current state ?